Laura content apr21 (#328)
saucelabs/sauce-docs: docs/secure-connections/sauce-connect/setup-configuration/high-availability.md

2. A service in Sauce Labs chooses a tunnel from the existing pool at random.
3. The resulting tunnel from step 2 is assigned to the new test VM.

### Basic Setup Using High Availability

<img src={useBaseUrl('img/sauce-connect/scp-basic-ha.png')} alt="Basic setup using High Availability" width="400"/>

## Multiple Network Routes
In this diagram, we see a setup that allows for multiple network routes when reaching the site(s) under test. There are variations to this, of course. For example, some of the Sauce Connect Proxy instances could be in the same network infrastructure as the site(s) under test. You can also imagine leveraging different virtual routes inside the same physical network. What is gained by setting up tunnels in this manner is redundancy for Sauce Connect Proxy tests, should one network route fail.

### Multiple Network Routes to Site Under Test (SUT)

<img src={useBaseUrl('img/sauce-connect/scp-mult-routes.png')} alt="Multiple network routes to Site Under Test (SUT)" width="400"/>

## High Availability Tunnel Settings and Commands

> You can customize your High Availability tests using these options below. For a full list of High Availability commands and other Sauce Connect Proxy options, see the [Sauce Connect Proxy Command Line Quick Reference Guide](https://wiki.saucelabs.com/display/DOCS/Sauce+Connect+Proxy+Command-Line+Quick+Reference+Guide).

Reviewer: ```suggestion
You can customize your High Availability tests using these options below. For a full list of High Availability commands and other Sauce Connect Proxy options, see the [Sauce Connect Proxy Command Line Quick Reference Guide](/dev/cli/sauce-connect-proxy).
```

Tunnel pools also abide by general fault tolerance rules. For example, if one tunnel becomes unavailable or shuts down (i.e., due to user shutdown, crash, maintenance, or network partition), you can configure your tunnels to be restored automatically while the test traffic is routed to another tunnel in the pool.

>**NOTE:** Tunnel pools do not self-heal (see [Self-management](https://en.wikipedia.org/wiki/Self-management_(computer_science)) for more information) or restart automatically. If a tunnel stops, you'll need to restart it manually. Other than restarting a stopped or failed tunnel, tunnel pools generally run automatically without user intervention. If you're running a high number of tunnels, you may want to bypass the pool and specify which tunnel to use directly.

When in High Availability mode, we recommend restarting Sauce Connect Proxy tunnels every 24 hours. The tunnel will stay open until all tests associated with it have completed. At the same time, once the shutdown command has been sent, the tunnel is marked inactive in the eyes of the pool, and no new jobs will use it.

> * Windows users: [Running as a Microsoft Windows Service](/secure-connections/sauce-connect/proxy-tunnels.md)

Reviewer: ```suggestion
* Windows users: [Running as a Microsoft Windows Service](/secure-connections/sauce-connect/proxy-tunnels)
```

> * Linux users: [Monitoring with Service Management Tools](/secure-connections/sauce-connect/proxy-tunnels.md)

Reviewer: ```suggestion
* Linux users: [Monitoring with Service Management Tools](/secure-connections/sauce-connect/proxy-tunnels)
```
